docker-compose.yml

```diff
@@ -1,13 +1,37 @@
-version: "3.9"
-
 services:
   api:
     build: .
-    command: python manage.py runserver 0.0.0.0:8080
+    command: >
+      sh -c "python manage.py migrate&&
+             python manage.py runserver 0.0.0.0:8000"
+    restart: always
     volumes:
       - .:/app
     ports:
       - "8021:8080"
     env_file:
       - .env
+    depends_on:
+      db:
+        condition: service_healthy
+
+  db:
+    image: postgres
+    restart: always
+    environment:
+      POSTGRES_USER: postgres
+      PGUSER: postgres
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
+    ports:
+      - "5432:5432"
+    volumes:
+      - db:/var/lib/postgresql/data
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready"]
```

Comment:
better but still we can do better. pg_isready doesn't guarantee DB is fully ready, it only checks if it accepts any connection. 
This one e.g. `["CMD-SHELL", "pg_isready -U postgres -d postgres"]` should try connecting to specific db, with specific user, the same as django will try